doc/about/update_notifications/index.md

```diff
@@ -0,0 +1,57 @@
+# Update Notifications {#about_update_notifications}
+
+[TOC]
+
+Outdated software is a huge issue in science workflows. Especially when the software is not part of the packaging
+provided by the operating system, it tends to be installed once and never updated.
+
+We provide update notifications as a convenient service to app users so that they immediately see when a new version is
+available. When we assume that an application is being run by its developer we nag the developer to also keep their
+checkout of the Sharg library up-to-date. This ensures new features get to app developers faster, applications have
+fewer bugs, and we receive fewer bug reports on outdated versions.
+
+## How it works
+
+Upon starting an application using the `sharg::argument_parser`, it is determined whether the app is built in release
+mode (we assume a user is running it) or in debug mode (we assume the developer is running it).
+
+We then send a web-request to our server with the following:
+
+ * The application name and its version
+ * The Sharg version that the application was built with
+ * The operating system type (Linux, macOS, Windows or BSD)
+ * The CPU type (32 or 64bit)
+
+But we send at most one request per day [we keep track of this by saving a timestamp to `.config/seqan/app_name`].
+
+If a newer app version is registered in our database (or can be automatically determined), we inform the user about it.
+If the app is running in debug mode and the Sharg version is outdated, we assume the user is the app developer and
+inform them about it.
```

Comment:
Do we check for newer versions even in debug mode? I'd expected that it is being first checked if it is debug or release. and only if it is release the internet is being contacted.

Reply:
No if it is in debug mode, we assume the developer is working on the app and we contact the internet to inform the developer of new seqan3 versions.

Reply:
I am assuming you meant "sharg versions" not "seqan3 versions". But yes I see.

I have checked the code. We also remind the developer(debug mode) to register a new version of his app if the app registered on the server is outdated.

Reply:
> I am assuming you meant "sharg versions" not "seqan3 versions". But yes I see.

Yes! sorry.

```suggestion
inform them about it. Additionally, we recommend to register your app with us 
if it is not already registered (support@seqan.de).
```

How about this?

Reply:
Yes, that is good!